rasa/server.py

```diff
@@ -217,14 +217,21 @@ def event_verbosity_parameter(
 ) -> Optional[DialogueStateTracker]:
```

Comment:
```suggestion
) -> DialogueStateTracker:
```
it raises if `tracker = None`